core/platform-common/src/main/java/org/sunbird/util/CloudStorageUtil.java

```diff
@@ -45,18 +45,20 @@ public static String getSignedUrl(
   private static IStorageService getStorageService(String storageType) {
     String storageKey = PropertiesCache.getInstance().getProperty(JsonKey.ACCOUNT_NAME);
     String storageSecret = PropertiesCache.getInstance().getProperty(JsonKey.ACCOUNT_KEY);
-    return getStorageService(storageType, storageKey, storageSecret);
+    scala.Option<String> storageEndpoint = scala.Option.apply(PropertiesCache.getInstance().getProperty(JsonKey.ACCOUNT_ENDPOINT));
+    scala.Option<String> storageRegion = scala.Option.apply("");
```

Comment:
cloud store sdk has defined this as an optional parameter. cloud store sdk will work only for home region for s3 compliant api. This is because the jclouds v2.5.0 used by cloud store sdk works that way. So even if the value is passed as "" , it will be evaluated as the home region of the tenancy. Thats the way oci s3 compatiblity works.